src/isPlainObj.js

```diff
@@ -1,10 +1,11 @@
-import _isObject from 'ramda/src/internal/_isObject';
 import { pipe, both, equals, toString, pathSatisfies, curryN } from 'ramda';
 
 import isNull from './isNull';
 import isObjLike from './isObjLike';
 import isFunction from './isFunction';
 
+const isObject = val =>
```

Comment:
IMHO using following composition should do the trick too. Just to be consistent with other changes in other PRs. This suggestion should be completely equivavelnt.
```suggestion
const isObject = pipe(type, identical('Object'));
```

Reply:
Sure thing, I like that. I think I did this one first or I probably would have gone with that the first time.